test/framework/resourceversion_helpers.go

```diff
@@ -48,7 +49,7 @@ func ValidateResourceVersionStable(ctx context.Context, proxy ClusterProxy, name
 	Consistently(func(g Gomega) {
 		objectsWithResourceVersion, err := getObjectsWithResourceVersion(ctx, proxy, namespace, ownerGraphFilterFunction)
 		g.Expect(err).ToNot(HaveOccurred())
-		g.Expect(objectsWithResourceVersion).To(Equal(previousResourceVersions))
+		g.Expect(objectsWithResourceVersion).To(Equal(previousResourceVersions), cmp.Diff(objectsWithResourceVersion, previousResourceVersions))
```

Comment:
Should we just use BeComparableTo instead? (it uses cmp under the hood)

Reply:
ok4me if the output is good enough then.